src/main/java/com/icthh/xm/tmf/ms/communication/service/mail/EmailTemplateService.java

```diff
@@ -67,13 +67,13 @@ public class EmailTemplateService {
     @SneakyThrows
     public RenderTemplateResponse renderEmailContent(RenderTemplateRequest renderTemplateRequest) {
         String tenantKey = tenantContextHolder.getTenantKey();
-        String renderedContent = processEmailTemplate(tenantKey, renderTemplateRequest.getContent(), renderTemplateRequest.getModel(), renderTemplateRequest.getLang());
+        String renderedContent = processEmailTemplate(tenantKey, renderTemplateRequest.getContent(), renderTemplateRequest.getModel(), renderTemplateRequest.getLang(), UUID.randomUUID().toString());
```

Comment:
and why uuid? why not the templatePath that when sending an email? it will render differently, but it should do the same

Reply:
Added templatePath